rust/actyx/ax/src/libp2p_streaming_response/tests/mod.rs

```diff
@@ -272,6 +259,7 @@ fn err_response_size() {
 
 #[test]
 fn err_request_size() {
+    crate::util::setup_logger();
```

Comment:
I added this to try out the logging: with the TestWriter, logging output (including panics, which we feed via `log_panic` into `log`, from which `tracing_log` takes it into `tracing`) will be properly captured by `libtest` and shown as the failure output of a test.

The original purpose of setting up a tracing Subscriber in tests is to enable debugging what does wrong in case of a test failure by using

    RUST_LOG=debug cargo test <some-test-name-prefix>